client.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/hashicorp/go-cleanhttp"
 )
 
+var (
+	// ErrInvalidStatus is returned when Grafana replies with an HTTP status code > 400.
+	ErrInvalidStatus = fmt.Errorf("invalid status")
```

Comment:
Consider to use a `errors.New("...")` for errors declaration

Reply:
I've considered it dozens of times in the years I've written go.